bot.py

```diff
@@ -0,0 +1,16 @@
+# Somebody once told me the world is gonna roll me
+# I ain't the sharpest tool in the shed
+# She was looking kind of dumb with her finger and her thumb
+# In the shape of an "L" on her forehead
+# Well the years start coming and they don't stop coming
+# Fed to the rules and I hit the ground running
+# Didn't make sense not to live for fun
+# Your brain gets smart but your head gets dumb
+# So much to do, so much to see
+# So what's wrong with taking the back streets?
+# You'll never know if you don't go
```

Comment:
There are other ways of finding out than physically being there. Maybe a friend could go in your stead and report back? This is stupid.

Reply:
Just went, can confirm there is nothing there.